packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -466,6 +466,17 @@ class ScrollPosition extends ViewportOffset {
   ScrollActivity get activity => _activity;
```

Comment:
I'm no longer using it, I've added an isScrolling getter to ScrollPosition, as well as listeners.